src/app/components/shell/account-popover/account-popover.vue

```diff
@@ -11,7 +11,7 @@
 				v-if="shouldShowNew"
 				class="notification-tag tag tag-highlight anim-fade-enter anim-fade-leave"
 			>
-				<translate>NEW</translate>
+				!
```

Comment:
I want it to be just a simple circle like we do in the channel cards.

Reply:
I think we should also just put a little dot next to the Stickers menu item below instead of saying "NEW"

Reply:
I changed it in a commit, once we get it to the point where you like it, I'll also apply it to the stickers menu item.